main.go

```diff
@@ -196,11 +202,15 @@ func main() {
 					case !t.Valid():
 						level.Error(logger).Log("msg", "token is invalid", "exp", t.Expiry.String())
 					default:
-						if err := ioutil.WriteFile(cfg.file, []byte(t.AccessToken), 0644); err != nil {
-							level.Error(logger).Log("msg", "failed to write token to disk", "err", err)
-						} else {
-							d = t.Expiry.Sub(time.Now()) - cfg.margin
+						if err := ioutil.WriteFile(cfg.tempFile, []byte(t.AccessToken), 0644); err != nil {
```

Comment:
`ioutil` is deprecated now, it will continue working but it might be a better idea to use `os.WriteFile` here.

Reply:
Sounds good 😊

Reply:
We still haven't upgraded to go1.16 so we're good for now.